proto/README.md

```diff
@@ -52,10 +55,14 @@ confidnece | - | float | 0 to 1 probability of the object classification.
 bbox | - | [BoundingBox](#boundingbox) | Bounding box of the object.
 velocity | m/s | Vector3 | XYZ velocity of the object.
 tracking_status | - | [TrackingStatus](https://github.com/seoulrobotics/sensr_proto/blob/master/type.proto) | tracking status.
+yaw_rate | - | float | Object rotation angle along the Z axis. (radian)
+time_offset | - | [Duration](https://developers.google.com/protocol-buffers/docs/reference/csharp/class/google/protobuf/well-known-types/duration) | Life time of the object.
```

Comment:
It is not the lifetime. It is the time difference of the object timestamp (of the object's algo node) and the earliest algo node timestamp. @young-seoulrobotics please correct me